js/queen.js

```diff
@@ -0,0 +1,117 @@
+import { Piece } from "./piece.js"
+
+/*
+* Represents a queen chess piece that is used with the Board class
+*/
+export class Queen extends Piece{
+    /**
+     * Instantiates a queen piece with the given position and piece ID.
+     * The piece belongs to the player with the given player ID.
+     * 
+     * @param {int} id Piece's ID.
+     * @param {int} rowPos Piece's row position.
+     * @param {int} colPos Piece's column position.
+     * @param {int} playerOwner
+     * 
+     */
+    constructor(id, rowPos, colPos, playerOwner) {
+        super();
+        this.id = id;
+        this.rowPos = rowPos;
+        this.colPos = colPos;
+        this.playerOwner = playerOwner;
+        this.unicodeChar = "&#9813;";
+    }
```

Comment:
You'll need to rename the `playerOwner` parameter and attribute to `playerId` in the `constructor()` since the attribute has been changed in the `Piece` class.

Reply:
The playerOwner paramter and attribute are both playerId now.